src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTY.java

```diff
@@ -99,6 +104,14 @@ public void vmStartEvent(VMStartEvent se)  {
         MessageOutput.lnprint("VM Started:");
     }
 
+/*[IF CRIU_SUPPORT]*/
+    @Override
+    public void vmRestoreEvent(VMRestoreEvent se)  {
```

Comment:
The name `se` might be appropriate for `VMStartEvent` above, but it seems wrong here.

Reply:
How about `re`?

Reply:
Should already be changed to use `re`.